zoom_sesh.py

```diff
@@ -6,73 +6,97 @@
 from timeit import default_timer as timer
 from html_maker import HtmlMaker
 import aslogging as logging
+import shutil
 
 COLAB_ROOT = '/content'
+ALUMNI_FILE = 'alumni.xlsx'
 
 if os.getcwd() == COLAB_ROOT:  # In Colab
   NAMES_DIR = os.path.join(COLAB_ROOT, 'alumni_shuffler', 'names')
 else:  # On local machine
   NAMES_DIR = './names'
   
-def import_names(dir):
-  name_files = [f for f in os.listdir(NAMES_DIR) if f.startswith('yob')]
+def import_random_names(dir):
+  name_files = [f for f in os.listdir(dir) if f.startswith('yob')]
   df = pd.read_csv(os.path.join(dir, name_files[0]))
   return df
 
-def make_fake_data(max_people=40):
-  df = import_names(NAMES_DIR)
+def make_fake_data(dir_name, max_people=40, overwrite=True):
+  df = import_random_names(NAMES_DIR)
   track_names = ['optics', 'semi', 'polymer', 'sensors']
   years = list(map(str, range(2013, 2020)))
   df.columns = ['name', 'year', 'track']
   df['track'] = [random.choice(track_names) for _ in range(len(df))]
   df['year'] = [random.choice(years) for _ in range(len(df))]
 
-  df = df.iloc[:max_people]  
   person_id = list(map(str,np.arange(max_people).tolist()))
 
-  for i in person_id:
-    df[i] = np.zeros(len(df))
-    df[i+"_cnsctv"] = np.zeros(len(df))
+  #df = ZoomSesh._create_tracking_cols(df.iloc[:max_people])
```

Comment:
I don't understand, how are the tracking columns being created rn?

Reply:
Oh... Are tracking columns being made when you read the names from excel?

Reply:
Nice catch... the tracking columns are being created in the ZoomSesh object here:
 https://github.com/nneibaue/alumni_shuffler/blob/09a1b8c09f74c38ef21a7f580d7735178e824a22/zoom_sesh.py#L72

The commented line shouldn't be there. I think that was part of a test I was doing and forgot to take it out. I pushed a new commit with it removed